### ISSUE TITLE
Include A11y Mode info on A11y page

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5289]
Information about Accessibility Mode and recommendations for when to use it, need to be added to the A11y information page.  

This work cannot be done until the Accessibility mode is present.
